src/Services/Testing/Testing.API/Application/Queries/Solutions/SolutionQueries.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Dapper;
+using Npgsql;
+using Testing.API.DTOs.Solutions;
+
+namespace Testing.API.Application.Queries.Solutions;
+
+public interface ISolutionQueries
+{
+    Task<SolutionResponse> GetSolutionAsync(Guid id);
+
+    Task<IEnumerable<SolutionResponse>> GetAllSolutionsAsync();
+}
+
+public class SolutionQueries : ISolutionQueries
```

Comment:
Class and its models must be in one folder.
I usually create inner folder Models